### ISSUE TITLE
"Band" property of collections: Rename "null" to "all"?

### ISSUE THREAD

**Issue by christophfriedrich** [Open-EO/openeo-processes#79]
Because essentially, "null" *means* "all" -- if renamed, it may be less confusing for users?

**Comment by christophfriedrich** [comment]
Not sure whether you understood what I meant; my idea was certainly a web editor issue. Because the idea was that `null` still means "all" internally, but that in the dropdown the *display text* of the entry with value=`null` is changed to "all" instead of the technical "null".

Your idea is to replace `null` with `"all"` (as string) generally? Could be good too 🤔 Especially this would void my concern from the second comment. And it would make the JSON code more understandable too, which certainly isn't bad either.

**Comment by m-mohr** [comment]
No, I meant that the process specifies what null means with a title in the schema. Code wise we don't need any change, but processes should explain better what they values mean. Similarly how it is done in the reducer function for the arguments.

**Comment by m-mohr** [comment]
After fixing some "null" type schemas in the processes, the situation is better for the data type chooser in the parameter editor.

ToDos for the Web Editor:
- Either show `null` in the title bar to have a better connection to descriptions that mention `null` or add the descriptive text back after the title has been selected ("This field is set to `null`").
- Show the title (in italic?) instead of null in the process boxes (bands: null => bands: All bands).
- In the data type chooser replace the "internal" data types such as `temporal-interval` with more user friendly titles (e.g. "Temporal Interval").
